src/fiat/drhook/drhook.c

```diff
@@ -2195,16 +2208,18 @@ process_options()
 
   tid = drhook_oml_get_thread_num();
 
-  int silent = 0;
   env = getenv("DR_HOOK_SILENT");
-  silent = env ? atoi(env) : silent;
+  if (env) {
+    opt_silent = atoi(env);
```

Comment:
It looks like the default for DR_HOOK_SILENT has changed, from "not-silent" to "silent", because the `static int opt_silent` variable is default initialised to 1. Is this intentional?

Reply:
Good catch! It wasn't intentional, I'll patch it now